tests/infer/test_jit.py

```diff
@@ -286,6 +290,8 @@ def guide():
     elbo = Elbo(max_iarange_nesting=0,
                 strict_enumeration_warning=any([enumerate1, enumerate2]),
                 num_particles=inner_particles)
+    if Elbo is JitTraceEnum_ELBO:
```

Comment:
This is the extra complexity I'd like to avoid (see above). Ideally we could simply
```py
elbo = Elbo(..., ignore_jit_warnings=True)
```
and non-jit implementations would simply ignore the flag.

Reply:
+1. Will add the flag to the base class itself.